lib/Jira.js

```diff
@@ -126,7 +126,13 @@ class Jira {
       console.log("createRfdAndLinkToRfc() with rfcIssueKey: " + rfcIssueKey);
 
       const rfcIssue = await this.retrieveRfcIssueInfo(rfcIssueKey);
-      const rfcIssueVersion = rfcIssue.fields.fixVersions[0].name;
+      let rfcIssueVersion
+      try {
+        rfcIssueVersion= rfcIssue.fields.fixVersions[0].name;
+      }
+      catch(err){
+         console.log("The RFC version is not defined. Please define a version in RFC and restart the pipeline.")
+      }
```

Comment:
Could you throw error here since you have the message for restarting pipeline.
Without throwing, the code continues.